scalyr_agent/builtin_monitors/journald_monitor.py

```diff
@@ -576,9 +580,16 @@ def format_msg(
         if extra_fields is not None:
             for field_name in sorted(extra_fields):
```

Comment:
Is there an easy way to perform this sort just once, instead of every time we go to write a log message out?  Such as, when we read in the configuration, we perform the sort then?

Reply:
The extra fields are gotten from each journal message so I don't think we have that option.

Reply:
Ok, never mind then.